### ISSUE TITLE
About diagramming tools

### ISSUE THREAD

**Issue by DuckWillow** [nus-cs2113-AY2122S1/forum#116]
My team suggested using 'drawio' to draw the diagram and screenshot the image into a .png file. 

In this case, our 'source' format will be in .xml, where we can open it using 'drawio'. Is it still an acceptable way to document?

**Comment by okkhoy** [comment]
Yup.. that should be fine. In the document, it is necessary to use the images anyway.
Same thing applies to other tools like PlantUML
